src/node.cc

```diff
@@ -2137,6 +2137,32 @@ void Hrtime(const FunctionCallbackInfo<Value>& args) {
   fields[2] = t % NANOS_PER_SEC;
 }
 
+// CPUUsage exposes libuv's uv_getrusage() this-process resource usage accessor,
+// to access ru_utime (user CPU time used) and ru_stime (system CPU time used),
+// which are uv_timeval_t structs (long tv_sec, long tv_usec) as tuples
+// similar to process.hrtime() values.  Only it's seconds/microseconds instead
+// of hrtime()'s seconds/nanoseconds.
```

Comment:
If we align more with `hrtime()`, perhaps it also makes sense to provide a "diff" function, like `hrtime()`, where if you pass a previous result of `cpuUsage()` into the function, it will return a "diff" of the times, instead of the current time.  For one of the primary use cases I'm considering, I'd be wanting a diff anyway, so this cuts down a little bit on the amount of code I'd be writing.